source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
                      <p>Watermark</p>
                      <p>Stamp</p>
                      <p>physMedium</p>-->
+                     <p>Encoding the exent and the dimensions of a source:</p>
+                     <p><specList>
+                        <specDesc key="extent"/>
+                     </specList></p>
+                     <p>The element <gi scheme="MEI">extent</gi> is used to express size in terms. That means that other than physical dimensions this can be done in units such as ‘number of pages’, ‘number of folios’, ‘records’, ‘bytes’, ‘physical components’, etc. For specifying the physical size of the material, for example, height and width, the use of the <gi scheme="MEI">dimensions</gi> element is recommended.</p>
```

Comment:
```suggestion
                     <p>The element <gi scheme="MEI">extent</gi> is used to express size in terms other than physical dimensions. This can be done in units such as ‘number of pages’, ‘number of folios’, ‘records’, ‘bytes’, ‘physical components’, etc. For specifying the physical size of the material, for example, height and width, the use of the <gi scheme="MEI">dimensions</gi> element is recommended.</p>
```